tests/manage/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -111,50 +116,54 @@ def test_non_ocs_taint_and_tolerations(self):
                 kind=constants.SUBSCRIPTION,
             )
             sub_obj.patch(params=param, format_type="merge")
+            logger.info(f"Successfully added toleration to {sub}")
 
         # Add tolerations to the ocsinitializations.ocs.openshift.io
         param = (
             '{"spec":  {"tolerations": '
             '[{"effect": "NoSchedule", "key": "xyz", "operator": "Equal", '
             '"value": "true"}]}}'
         )
-
         ocsini_obj = ocp.OCP(
             resource_name=constants.OCSINIT,
             namespace=defaults.ROOK_CLUSTER_NAMESPACE,
             kind=constants.OCSINITIALIZATION,
         )
         ocsini_obj.patch(params=param, format_type="merge")
+        logger.info(f"Successfully added toleration to {ocsini_obj.kind}")
 
         # Add tolerations to the configmap rook-ceph-operator-config
         configmap_obj = ocp.OCP(
             kind=constants.CONFIGMAP,
             namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
             resource_name=constants.ROOK_OPERATOR_CONFIGMAP,
         )
-        toleration = configmap_obj.get().get("data").get("CSI_PLUGIN_TOLERATIONS")
-        toleration += (
+        toleration = (
             '\n- key: xyz\n  operator: Equal\n  value: "true"\n  effect: NoSchedule'
         )
         toleration = toleration.replace('"', '\\"').replace("\n", "\\n")
-        param_cmd = (
-            f'[{{"op": "replace", "path": "/data/CSI_PLUGIN_TOLERATIONS", "value": "{toleration}" }}, '
-            f'{{"op": "replace", "path": "/data/CSI_PROVISIONER_TOLERATIONS", "value": "{toleration}" }}]'
+
+        params = (
+            f'{{"data": {{"CSI_PLUGIN_TOLERATIONS": "{toleration}", '
+            f'"CSI_PROVISIONER_TOLERATIONS": "{toleration}"}}}}'
         )
-        configmap_obj.patch(params=param_cmd, format_type="json")
+
+        configmap_obj.patch(params=params, format_type="merge")
+        logger.info(f"Successfully added toleration to {configmap_obj.kind}")
 
         # After edit noticed few pod respins as expected
-        assert wait_for_pods_to_be_running(timeout=600, sleep=15)
+        assert wait_for_pods_to_be_running(timeout=900, sleep=15)
 
-        # Respin all pods and check it if is still running
-        pod_list = get_all_pods(
-            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
-        )
-        for pod in pod_list:
-            pod.delete(wait=False)
+        # Check non ocs toleration on all pods under openshift-storage
+        check_toleration_on_pods(toleration_key="xyz")
 
-        assert wait_for_pods_to_be_running(timeout=600, sleep=15)
-        self.sanity_helpers.health_check()
```

Comment:
why is sanity_helpers removed from the test ?

Reply:
Added sanity_helpers